examples/enable/basic_move.py

```diff
@@ -3,10 +3,11 @@
 """
 from __future__ import print_function
```

Comment:
I think we're at a place where we can get rid of all of the `__future__` imports and other python 2/3 compatibility code in the examples. What do you think?

Reply:
Should do it as part of a separate PR